tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -242,14 +243,19 @@ void onRenderTaskCancelled() {
 
   void onRenderTaskPostExecute() {
     mIsRendering = false;
-    if( !mTransitionsEnabled ) {
-      cleanup();
-    }
-    if( mTileRenderListener != null ) {
-      mTileRenderListener.onRenderComplete();
-    }
-    invalidate();
-    requestRender();
+    mTileRenderHandler.post( new Runnable() {
```

Comment:
IIUC, sounds like we're all on the same page.  So: A in the first example, and B in the second example?